app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
##### UsersController

##### `new` action

* [ ] **_It should have a `new` action with `User.new` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_up_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49722686/))_**

##### `create` action

* [ ] **_It should have a create action. Must use `user_params` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_up_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49722686/))_**

Reply:
Hi guys, we missed this section. Users controller must have a new and create action as pointed out by @lekegitrepo. We currently have new method which should be rename as `create` and create a new method like this 
``` 
def new
  @user = User.new
end
```